monokai-theme.el

```diff
@@ -4610,6 +4610,17 @@ Also affects 'linum-mode' background."
                                         :background ,monokai-256-background
                                         :inverse-video t))))
 
+   ;; realgud
+   `(realgud-overlay-arrow1
+     ((,monokai-class (:foreground ,monokai-green-d))
+      (,monokai-256-class  (:foreground ,monokai-green-d))))
```

Comment:
Thanks for looking this over. (I make a lot of mistakes which is why I work on debuggers.) Ths should be addressed in commit  2e89bf7 . However I've added more realgud definitions. 

And speaking of mistakes... 

In 
https://github.com/oneKelvinSmith/monokai-emacs/blob/master/monokai-theme.el#L679
shouldn't that be _source_ instead of "soure"?